stock_mts_mto_rule/README.rst

```diff
@@ -0,0 +1,92 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
It would be nice to split the readme, like explain here : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0